src/os/bluestore/BlueStore.cc

```diff
@@ -4395,6 +4411,31 @@ bool BlueStore::is_rotational()
   return rotational;
 }
 
+bool BlueStore::is_journal_rotational()
+{
```

Comment:
@liewegas I used the above implementation and I am hitting `FAILED assert(bluefs)`. 
Looking at logs, I noticed that bluestore umount() and bluefs umount() are being called before `BlueStore::is_journal_rotational()`. I'll dig into this further to figure out what's going on. Any ideas?